tutorial/20_recipes/006_user_defined_pruner.py

```diff
@@ -43,10 +44,10 @@
 implement a pruner which terminates a trial at a certain step if it is in last
 place compared to completed trials at the same step. We begin considering
 pruning after a "warmup" of 1 training step and 5 completed trials. For
-demonstration purposes, we print() a diagnostic message from prune() when it is
-about to return True (indicating pruning).
+demonstration purposes, we :func:`print` a diagnostic message from ``prune`` when
```

Comment:
I want to know why ``prune`` here is not documented with an rst method syntax ":func:`~optuna.xxx.prune`".

Reply:
The prune refers to the one implemented in the `LastPlacePruner`.
Therefore I don't think :func:~optuna.xxx.prune is appropriate in the line.

Reply:
I agree with @crcrpar; this is why it isn't documented.